public/components/stub-modal/stub-modal.html

```diff
@@ -67,14 +67,6 @@ <h4 class="modal-title">{{ modalTitle }}</h4>
                 </div>
             </div>
         </div>
-
-    </div>
-    <div class="modal-body modal-body__header" ng-click="expanded = !expanded">
```

Comment:
Just to double check, is this 'expanded' variable set in the scope anywhere? Does it need to be removed?

(It's possible to set this in the template only, so it may be this is the only reference)

Reply:
As far as I can see, this is the only place it's set or referenced. There's no mention of it in the corresponding JS file.